bigquery/deploy/deploy-indexer.sh

```diff
@@ -40,5 +40,6 @@ docker push gcr.io/${project_id}/bq-indexer
 cd deploy
 kubectl delete configmap dataset-config
 kubectl create configmap dataset-config --from-file=../../dataset_config/${dataset}
-kubectl delete -f bq-indexer.yaml
+# Ignore errors in deleting the bq-indexer in case it does not yet exist.
```

Comment:
Somehow incorporate "set -o errexit" into this comment.

Can you add this comment to deploy.sh also? Thanks.